src/CommandingOfficers/AWBW/IDS/TabithaBW.java

```diff
@@ -0,0 +1,136 @@
+package CommandingOfficers.AWBW.IDS;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderAbility;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.DeployableCommander;
+import CommandingOfficers.MeteorParams;
+import CommandingOfficers.AW4.IDS.Tabitha.FirestormValueFinder;
+import CommandingOfficers.AWBW.AWBWCommander;
+import CommandingOfficers.AWBW.COUableCommander;
+import Engine.GameScenario;
+import Engine.XYCoord;
+import Engine.Combat.DamagePopup;
+import Engine.GameEvents.GameEvent;
+import Engine.GameEvents.GameEventQueue;
+import Terrain.GameMap;
+import Terrain.MapMaster;
+import UI.UIUtils;
+import Units.Unit;
+
+public class TabithaBW extends COUableCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Tabitha", UIUtils.SourceGames.AWBW, UIUtils.IDS);
+      infoPages.add(new InfoPage(
+          "Can grant a single Mega Boost of +"+POWER+"/"+DEFENSE+" stats.\n"
+        + "This powerup lasts until the unit is destroyed.\n"));
+      infoPages.add(new InfoPage(
+          "Firestorm (6):\n"
+        + "4 HP strike that targets HP with gunboats counting half.\n"
```

Comment:
HP strike targets HP, huh? Seems odd to call out (twice) that strikes can cause HP damage.

Also, "gunboats counting half" - as in they take half damage, or something else?

Reply:
It _targets_ HP instead of funds value.
Gunboats again, count as half (HP, IIRC) value for targeting.

Reply:
Or, reparsing this, I'm guessing it is an AOE attack that tries to inflict the maximum amount of overall HP damage (or maybe it tries to target units with the most HP? not necessarily the same), but counts gunboats as less important. If so, verbiage could be clearer, and also list the strike's AOE range.